src/components/RangeSlider/components/Labelled.tsx

```diff
@@ -0,0 +1,88 @@
+import *as React from 'react';
+import { buttonFrom } from '../../Button';
+import Label, { labelID } from '../../Label';
+
+export { labelID };
+
+type Falsy = boolean | undefined | null | 0;
+
+export function classNames(...classes: (string | Falsy)[]) {
+  return classes.filter(Boolean).join(' ');
+}
+
+export function variationName(name: string, value: string) {
+  return `${name}${value.charAt(0).toUpperCase()}${value.slice(1)}`;
+}
+
+export interface LabelProps {
+  children?: string;
+  id: string;
+  hidden?: boolean;
+}
+
+export interface LabelledProps {
+  id: LabelProps['id'];
+  label: string;
+  error?: any;
+  action?: any;
+  helpText?: React.ReactNode;
+  children?: React.ReactNode;
+  labelHidden?: boolean;
+}
+
+// tslint:disable-next-line: function-name
+export function Labelled({
+  id,
+  label,
+  error,
+  action,
+  helpText,
+  children,
+  labelHidden,
+  ...rest
+}: LabelledProps) {
+  const className = classNames(labelHidden);
+
+  const actionMarkup = action ? (
+    <div >{buttonFrom(action, { plain: true })}</div>
+  ) : null;
+
+  const helpTextMarkup = helpText ? (
+    <div  id={helpTextID(id)}>
+      {helpText}
+    </div>
+  ) : null;
+
+  const errorMarkup = error && typeof error !== 'boolean' && (
+    <div >
+      {/* <InlineError message={error} fieldID={id} /> */}
```

Comment:
Do we need this commented code?

Reply:
Yes, we need this commented code, It is not commented now.